src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,85 @@
+package duke;
+import java.util.Scanner;
+
+public class Duke {
+
+    Scanner in = new Scanner(System.in);
+    private String commands;
+    Task[] tasks = new Task[100];
+
+    Duke() {
+        System.out.println("Hello, I am Duke!");
+        System.out.println("What can I do for you?");
+    }
+
+    void processCommands(){
+        int i = 0;
```

Comment:
Should the variable 'i' be renamed to something more descriptive?
example: totalTaskAdded